components/vies_api/vies_api.app.mjs

```diff
@@ -1,11 +1,14 @@
+import VIESAPI from "viesapi-client";
```

Comment:
The recommended is not to use axios instead of a specific lib?

That's just a question, so I will move forward to QA.

Cc: @malexanderlim